component/service/src/main/java/org/exoplatform/social/rest/impl/portletBanner/PortletBannerRestService.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (C) 2023 eXo Platform SAS.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.exoplatform.social.rest.impl.portletBanner;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import org.apache.commons.lang3.StringUtils;
+import org.exoplatform.social.core.model.BannerAttachment;
+import org.exoplatform.social.core.portletBanner.PortletBannerService;
+import org.exoplatform.services.log.ExoLogger;
+import org.exoplatform.services.log.Log;
+import org.exoplatform.services.rest.resource.ResourceContainer;
+import org.exoplatform.services.security.ConversationState;
+import org.exoplatform.services.security.Identity;
+import org.exoplatform.social.core.service.LinkProvider;
+import org.exoplatform.social.rest.entity.PortletBannerEntity;
+import org.exoplatform.social.service.rest.api.VersionResources;
+
+import javax.annotation.security.RolesAllowed;
+import javax.ws.rs.*;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Request;
+import javax.ws.rs.core.Response;
+import java.io.InputStream;
+
+@Path(VersionResources.VERSION_ONE + "/social/portletBanner")
+@Tag(name = VersionResources.VERSION_ONE + "/social/portletBanner", description = "Managing portlet Banner")
```

Comment:
```suggestion
@Path(VersionResources.VERSION_ONE + "/social/banner")
@Tag(name = VersionResources.VERSION_ONE + "/social/banner", description = "Managing banner")
```
We should check if /social is adequate with the URL